src/bb-components-build.ts

```diff
@@ -1,14 +1,18 @@
 // tslint:disable:no-console
 // tslint:disable:no-new-func
-// tslint:disable:function-name
 import program, { CommanderStatic } from 'commander';
 import { promises, outputJson, pathExists } from 'fs-extra';
-import { ComponentProps, PrefabProps } from './types';
-import { getDuplicateNames, checkRequiredProps } from './utils/validation';
+import { Component, Prefab, Partial } from './types';
+import { validateDuplicateNames } from './utils/validation';
+
+import { validateSchema as validateComponentSchema } from './validations/component';
+import { validateSchema as validatePrefabSchema } from './validations/prefab';
+import { validateSchema as validatePartialSchema } from './validations/partial';
```

Comment:
Is export default possible?

Reply:
We will move some of the other validations (e.g. checking names) to these modules. Also, callers can use `import * as Component from './component'`. So I guess we shouldn't use export default in this case.